src/ephemeral.rs

```diff
@@ -537,14 +542,15 @@ pub(crate) async fn start_ephemeral_timers(context: &Context) -> sql::Result<()>
 
 #[cfg(test)]
 mod tests {
+    use crate::param::Params;
     use async_std::task::sleep;
 
     use super::*;
-    use crate::test_utils::TestContext;
     use crate::{
         chat::{self, Chat, ChatItem},
         dc_tools::IsNoneOrEmpty,
     };
+    use crate::{config::Config, test_utils::TestContext};
```

Comment:
`Config` is not used

Reply:
Do you know why the CI didn't fail? (https://github.com/deltachat/deltachat-core-rust/runs/2410685416 or https://github.com/deltachat/deltachat-core-rust/runs/2410628627)

Reply:
It's not a clippy warning, but `rustc` warning. There are annotations, but it does not fail the job.

We could add `RUSTFLAGS=-Dwarnings` to `cargo check` environment.

Reply:
Ah, ok.